Userland/Libraries/LibGfx/ImageFormats/GIFWriter.cpp

```diff
@@ -155,6 +188,7 @@ ErrorOr<void> GIFAnimationWriter::add_frame(Bitmap& bitmap, int duration_ms, Int
 
     // Write a Table-Based Image
     BigEndianOutputBitStream bit_stream { MaybeOwned { m_stream } };
+    TRY(write_graphic_control_extension(bit_stream, duration_ms));
```

Comment:
nit: commit description says "this is quite useless until we write animations", but we already do

Reply:
Ah, didn't check that, updated!